### ISSUE TITLE
HTTP requests to have POST + extra data

### ISSUE THREAD

**Issue by gfwilliams** [gfwilliams/Gadgetbridge#5]
Requested at http://forum.espruino.com/conversations/378303/#comment16626919

Shouldn't be too painful to add...

**Comment by gfwilliams** [comment]
Currently we have:

```
t:"htt­p", url:"..."[,xpath:"­..."][,id:"..."]
```

So something like:

```
t:"htt­p", url:"...", method:"POST", body:"...", headers:{header:content}
```

could be good - and then it'd follow http://www.espruino.com/Internet#http-post quite well

**Comment by gfwilliams** [comment]
fixed in 4cd53fa9b64fa668960e27b03fed07edb7237240
